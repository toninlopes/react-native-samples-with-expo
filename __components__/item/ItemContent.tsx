import React from "react";
import { Text, StyleSheet, View } from "react-native";
import type { ItemContentProps } from './types';

const ItemContent: React.FC<ItemContentProps> = ({ title }) => {
  return (
    <View style={styles.view}>
      <Text style={styles.title}>{title}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  view: {
    flex: 1,
  },
  title: {
      fontSize: 22,
  },
});

export default ItemContent;