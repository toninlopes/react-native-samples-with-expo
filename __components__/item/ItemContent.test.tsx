import React from 'react';
import renderer from 'react-test-renderer';
import ItemContent from './ItemContent';

describe('ItemContent', () => {
  test('When there is title.', () => {
    const itemContent = renderer.create(<ItemContent title='Title' />).toJSON();
    expect(itemContent).toMatchSnapshot();
  });
});