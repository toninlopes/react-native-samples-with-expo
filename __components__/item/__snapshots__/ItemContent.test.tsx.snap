// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemContent When there is title. 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <Text
    style={
      {
        "fontSize": 22,
      }
    }
  >
    Title
  </Text>
</View>
`;
